www/js/app.js

```diff
@@ -1017,7 +1038,7 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         // but we should not do this when opening the landing page (or else one of the Unit Tests fails, at least on Chrome 58)
         if (!params.isLandingPage) document.getElementById('articleContent').contentWindow.focus();
 
-        if (contentInjectionMode === 'serviceworker') {
+        if (contentInjectionMode === '1') {
```

Comment:
Why this change??
It looks like a mistake that breaks the ServiceWorker mode.

Please review your code before asking for a review from someone else.